styles/style.css

```diff
@@ -0,0 +1,120 @@
+* {
+    font-family: Arial;
+}
+
+
+nav {
+    width: 100%;
+    margin: 0 auto;
+    position: fixed;
+    background-color: #4ABDAC;
+    top: 0;
+    left: 0;
+
+}
+
+
+header, header ul li a{
+    height: 60px;
+    color: #E9E9E9;
+    font-size: 25px;
+}
+
+
+
+header p {
+    /*margin-top: 0;*/
+    margin-left: 60px;
+    display: inline-block;
+    margin-bottom: 0;
+}
+
+header nav ul {
+    list-style: none;
+    display: inline-block;
+    float: right;
+    clear: both;
+    margin-bottom: 0px
+}
+
+header nav ul li {
```

Comment:
This is a perfect place to put an id on the `<li>` tag. Call it `nav-links` and use that in your CSS. Make it an obvious name, and we won't have to look for each element in turn. It makes sense that navbar list items are special - use an ID to show that.